src/cdk/tree/nested-node.ts

```diff
@@ -53,15 +53,16 @@ import {getTreeControlFunctionsMissingError} from './tree-errors';
   },
   providers: [{provide: CdkTreeNode, useExisting: CdkNestedTreeNode}]
 })
-export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements AfterContentInit, OnDestroy {
+export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements
+    AfterContentChecked, AfterContentInit, OnDestroy {
   /** Differ used to find the changes in the data provided by the data source. */
   private _dataDiffer: IterableDiffer<T>;
 
   /** The children data dataNodes of current node. They will be placed in `CdkTreeNodeOutlet`. */
   protected _children: T[];
 
   /** The children node placeholder. */
-  @ContentChildren(CdkTreeNodeOutlet) nodeOutlet: QueryList<CdkTreeNodeOutlet>;
+  @ContentChild(CdkTreeNodeOutlet) nodeOutlet: CdkTreeNodeOutlet;
```

Comment:
This is a breaking change since the property isn't underscored and the type has changed.

Reply:
I didn't think about that.  Guess I will not worry about trying to remove the `ContentChildren` usage.